### PR TITLE
fix(alertmanager-integrations): decouple slack route name from channel: fix webhook url secret ref

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: alerts/charts/templates/am-base-config.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
         value: "Watchdog|InfoInhibitor"
     routes:
       {{- range $route := .Values.alerts.alertmanagerConfig.slack.routes }}
-      - receiver: {{ required ".Values.alerts.alertmanagerConfig.slack.routes[].name missing" $route.channel }}
+      - receiver: {{ required ".Values.alerts.alertmanagerConfig.slack.routes[].name missing" $route.name }}
```
[the remark this answers is not in the document]
Comment (richardtief) [reply alerts/charts/templates/am-base-config.yaml:19]: Thanks! Fixed.

File: alerts/charts/values.yaml [hunk 2]
```diff
@@ -145,7 +145,7 @@ alerts:
       route:
         group_by:
           - organization
-          - alert_name
+          - alertname
```
Comment (viennaa) [inline alerts/charts/values.yaml:148]:
```suggestion
          - alert_name
```

ref: #95

Comment (richardtief) [reply alerts/charts/values.yaml:148]: Yeah, this was wrong with _

**Review by viennaa — APPROVED** [review]
Thanks and let's go.

**Pull request merged**